This is just for a custom build for a customer.

BramGruneir opened the pull request:
For a custom build, do not merge.

Commit:
importccl: add ignore lines option to DELIMITED

This is indeed when an input has a header that needs to be
ignored from importing. The new option is used as:
`IMPORT ... DELIMITED DATA ... WITH skip = 'n';`
where n is the number of lines to be ignored at beginning of file.

Release note (sql): add new option skip to DELIMITED.

Commit:
importccl: add nullif option to DELIMITED

This is indeed when an input has a field that needs to
be treated as NULL. Example:
```
import table t022 (
a int, b string
) DELIMITED DATA ('nodelocal:///foo.csv') with nullif = '';
```
This will force all empty string fields to be inserted as
NULL in the table.

Release note (sql): add nullif option to IMPORT ... DELIMITED.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/41525)
<!-- Reviewable:end -->

BramGruneir commented:
@spaskob Here's my pr/branch with your changes.

spaskob commented:
Thanks, any AIs for me?

On Thu, Oct 10, 2019 at 8:38 PM Bram Gruneir <notifications@github.com>
wrote:

> @spaskob <https://github.com/spaskob> Here's my pr/branch with your
> changes.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/41525?email_source=notifications&email_token=AANL6365JRL3TEJC6UAJ5I3QN7DGTA5CNFSM4I7TJRNKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEA6MD2I#issuecomment-540852713>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANL634J3BV47NQF654EB7TQN7DGTANCNFSM4I7TJRNA>
> .
>

BramGruneir commented:
nope, all good

Commit:
importccl: add strict_quotes option to CSV

LazyQuotes is an option on the Go csv reader. We switched it
off for 19.2 but this breaks backward compatibility for clients
that relied on this option to parse lazily quotes in csv files.
With this PR we revert back to LazyQuotes being true by default
and we introduce an `WITH strict_quotes` option for `CSV IMPORT`
to allow to switch off LazyQuotes. Note that we'd like to keep
this undocumented for now since the LazyQuotes mode is not very
well understood.

Release notes (sql change): add strict_quotes option to CSV.
